src/ch.usi.inf.nodeprof/src/ch/usi/inf/nodeprof/handlers/BaseEventHandlerNode.java

```diff
@@ -181,34 +183,45 @@ private void reportAttributeMissingError(String key, Exception e) {
      *
      * @param index
      * @param inputs
-     * @param inputName
+     * @param inputHint
      * @return the value of inputs[index]
      */
-    protected Object assertGetInput(int index, Object[] inputs, String inputName) {
+    protected Object assertGetInput(int index, Object[] inputs, String inputHint) {
         if (inputs == null) {
-            reportInputsError(index, inputs, "InputsArrayNull");
+            reportInputsError(index, null, "InputsArrayNull", inputHint);
```

Comment:
why `null`?

Reply:
This case here is to catch potential loss of inputs in the instrumentation. It was introduced long ago for debugging purposes.

Reply:
ah OK thanks